DDR4 OC Guide.md

```diff
@@ -420,6 +420,8 @@ Some terminology:
   | 1000 | 65 - 75 |
   | 2000 | 60 - 70 |
   | 3000 | 65 - 75 (1:1 MCLK:FCLK) <br/> 75+ (2:1 MCLK:FCLK) |
+  | 4000/5000G | ??? |
+  | 5000 | 60 - 70 (1:1 MCLK:FCLK) <br/> 70+ (desynchronised FCLK) |
```

Comment:
With B-die and 1900 FCLK, ~55ns is possible IIRC but for most people this range is about right

Reply:
Certain Ryzen 5000s can actually do 2000 FCLK even